specs/vm/instruction_set.md

```diff
@@ -1785,3 +1786,101 @@ Panic if:
 - `$fp->$fp == 0` (if parent context is external)
 
 Set `$rA` to `$fp->$fp` (i.e. `$rA` will point to the previous call frame's contract ID).
+
+### GT: Get transaction fields
```

Comment:
So we already have an instruction called `gt` (greater than). Should we call this one `gtf` instead?

Reply:
🤦 Whoops.

Reply:
Fixed in [4813ff0](https://github.com/FuelLabs/fuel-specs/pull/353/commits/4813ff0bc852b8af208b38d3ff791a7b60d4d1ff)